app/controllers/api/api_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Api
+  class ApiController < ApplicationController
+    skip_before_action :verify_authenticity_token
+
+    def posts_index
+      @user = User.find_by_api_token(request.headers['X-AUTH-TOKEN'])
+      @posts = Post.where(user_id: @user.id)
+      render json: @posts
+    end
+
+    def list_comments
+      @user = User.find_by_api_token(request.headers['X-AUTH-TOKEN'])
+      @comments = Comment.where(user_id: @user_id)
+      render json: @comments
+    end
+
+    def create_comment
+      @user = User.find_by_api_token(request.headers['X-AUTH-TOKEN'])
+      @new_comment = @user.comments.new(post_id: params[:post_id], text: params[:text])
+
+      if @new_comment.save
+        render json: @new_comment, status: :created
+      else
+        render json: @new_comment.errors, status: :unprocessable_entity
+      end
+    end
+
+  end
```

Comment:
![image](https://user-images.githubusercontent.com/55635977/162442723-3fcaebb6-7561-46e9-bb4d-a8e8e0b39385.png)

![image](https://user-images.githubusercontent.com/55635977/162442885-9b5a9178-53c1-48b0-a5a0-06718ddbc98c.png)

Reply:
Thank you @MrBrN197 and [Emmyn5600](https://github.com/Emmyn5600). You just made our day!